.travis.yml

```diff
@@ -49,9 +52,16 @@ matrix:
       env: DB=pgsql POSTGRESQL_VERSION=9.2 # driver currently unsupported by HHVM
     - php: hhvm
       env: DB=pgsql POSTGRESQL_VERSION=9.3 # driver currently unsupported by HHVM
+    - php: hhvm
+      env: DB=pgsql POSTGRESQL_VERSION=9.4 # driver currently unsupported by HHVM
```

Comment:
I wondered that myself. I would imagine this is no longer the case, but I
don't use hhvm.
On Mar 31, 2015 3:16 AM, "Marco Pivetta" <notifications@github.com> wrote:

> In .travis.yml
> <https://github.com/doctrine/dbal/pull/825#discussion_r27457070>:
>
> > @@ -49,9 +52,16 @@ matrix:
> >        env: DB=pgsql POSTGRESQL_VERSION=9.2 # driver currently unsupported by HHVM
> >      - php: hhvm
> >        env: DB=pgsql POSTGRESQL_VERSION=9.3 # driver currently unsupported by HHVM
> > +    - php: hhvm
> > +      env: DB=pgsql POSTGRESQL_VERSION=9.4 # driver currently unsupported by HHVM
>
> Wondering if this still holds true considering
> https://github.com/PocketRent/hhvm-pgsql. Is HHVM really not supporting a
> single actually working RDBMS?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/doctrine/dbal/pull/825/files#r27457070>.
>